### PR TITLE
Neural Network Update README.md

### PR DESCRIPTION
**Pull request by aanchalsethi**

### CODE REVIEW

File: Neural Network/README.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+## Artificial Neuron
+
+![image](https://user-images.githubusercontent.com/67017422/115409740-a6b4ad80-a20f-11eb-9abe-eb7c31af1af9.png)
+
+
+Artificial neurons are the basic unit of a neural network. The artificial neuron takes one or more inputs and adds them together to create an output. Perceptrons are another name for artificial neurons. An artificial neuron is:
+
+Y= Σ (weights * input) + bias
+
+wights= It controls the signal between two neurons (or the intensity of the connection) To put it another way, a weight determines how much of an impact the input has on the output.
```
Comment (ashwani-rathee) [inline Neural Network/README.md:24]:
```suggestion
weights = It controls the signal between two neurons (or the intensity of the connection). To put it another way, a weight determines how much of an impact the input has on the output.
```

File: Neural Network/README.md [hunk 2]
```diff
@@ -1 +1,75 @@
...
+
+Bias= Constant biases are an extra input into the next layer that often has the value of one. The bias unit ensures that even though all of the inputs are zeros, the neuron will still be activated.
```
Comment (ashwani-rathee) [inline Neural Network/README.md:26]: Bias definition is not clear and I am not sure why it's value is often one

**Review by ashwani-rathee — APPROVED** [review]
Good work there

**Review by geekquad — APPROVED** [review]
Hey @aanchalsethi, 
amazing work!

**Pull request merged**